skeleton/template.yaml

```diff
@@ -253,6 +253,13 @@ spec:
       input:
         url: ./content
         targetPath: source
+    # Renders all of the template variables into the techdocs and adds them to the source repo
+    - id: fetch-skeleton-docs
```

Comment:
Since we are introducing this step, does `fetch-base` not need the `input.values`?

Reply:
Yeah that's correct, the values from `fetch-base` were being used to fill in variables in the docs since they would be in the `/content` directory that `fetch-base` grabs. When moving the docs out to their own action the values move with it if that makes sense